From: Simson Garfinkel <simsong@acm.org>
Date: Sun, 16 Feb 2014 13:48:41 -0500
Subject: fixed handling of fputc
Forwarded: yes. merged upstream at https://github.com/simsong/tcpflow/commit/1aa7d8e5072175859b0a3c41264c84de14b25e55
---
 src/tcpip.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tcpip.cpp b/src/tcpip.cpp
index ecad602..86ad6df 100644
--- a/src/tcpip.cpp
+++ b/src/tcpip.cpp
@@ -268,7 +268,7 @@ void tcpip::print_packet(const u_char *data, uint32_t length)
                     exit(1);
                 
                 }
-                written += ret;
+                written += 1;
 	    }
 	}
     }
